build-system/tasks/visual-diff/index.js

```diff
@@ -159,7 +159,7 @@ async function launchWebServer() {
   await startServer(
     {host: HOST, port: PORT},
     {quiet: !argv.webserver_debug},
-    {compiled: argv.compiled}
+    {compiled: true}
```

Comment:
No actual objections to this change, but is there a use case where it might be useful to run visual tests on minified code?

For comparison: E2E and integration tests allow usage with and without `--compiled`, while performance tests and coverage map tests assume minified code and don't support the `--compiled` flag.

Reply:
I think right now it doesn't _really_ work with unminified code, because it verifies the existence of `dist/v0.js` - so for now I'd rather remove this option